docs/book/src/topics/windows.md

```diff
@@ -6,7 +6,7 @@ CAPZ enables you to create Windows Kubernetes clusters on Microsoft Azure. We re
 
 ### Using Containerd for Windows Clusters
 
-To deploy a cluster using Windows, use the [Windows Containerd flavor template](../../../../templates/cluster-template-machinepool-windows-containerd.yaml).
+To deploy a cluster using Windows, use the [Windows Containerd flavor template](https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/cluster-template-windows-containerd.yaml).
```

Comment:
@jackfrancis do we need to update this when #2315 merges?

Reply:
yep, it was always going to be a race between this PR and #2292

Reply:
ok since #2292 is about to merge, @priyal13m can you please make the following change?

```suggestion
To deploy a cluster using Windows, use the [Windows Containerd flavor template](https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/main/templates/cluster-template-windows.yaml).
```